src/db/spanner/models.rs

```diff
@@ -137,6 +137,13 @@ impl SpannerDb {
         }
     }
 
+    pub(super) fn get_collection_name(&self, id: i32) -> String {
+        self.coll_cache
+            .get_name(id)
```

Comment:
this returns `<Result<Option>>`

we probably wanna propagate it, there's a lock underneath which could have been poisoned. feels bad hiding that kind of error

```suggestion
            .get_name(id)?
```

Reply:
(FYI I actually implemented a _get_collection_name for mysql. apparently it is never used)

Reply:
so you're saying that it's a TODO to make this a trait? ;)